src/modules/UI/selectors.js

```diff
@@ -47,12 +46,12 @@ export const getDefaultDenomination = (state: RootState, currencyCode: string):
   const settings = state.ui.settings
   const currencySettings = settings[currencyCode]
   const defaultMultiplier = currencySettings.denomination
-  const denomination = _.find(currencySettings.denominations, denom => denom.multiplier === defaultMultiplier)
+  const denomination = currencySettings.denominations.find(denom => denom.multiplier === defaultMultiplier)
   if (!denomination) throw new Error('Edge: Denomination not found. Possible invalid currencyCode.')
   return denomination
 }
 
-export const getExchangeDenomination = (state: RootState, currencyCode: string, specificWallet?: GuiWallet): GuiDenomination => {
+export const getTokenExchangeDenomination = (state: RootState, currencyCode: string, specificWallet?: GuiWallet): GuiDenomination => {
```

Comment:
Why did you change the name?
If I understand correctly, this function returns the denom for a mainnet currency's OR a token's one.

Reply:
The problem is that there are two functions with the same `getExchangeDenomination` name, one in `UI/selectors.js` and another in `settings/selectors.js`. This one is aware or tokens, and the other doesn't seem to be, so I added "token" to the name.

Reply:
Please advise.

Reply:
I changed the name to `getPrimaryExchangeDenomination` because every single place it is used has the shape `const primaryExchangeDenomination = getPrimaryExchangeDenomination(...)`. Since the left-hand side always has the name `primaryExchangeDenomination`, that's what it should probably be called.